src/wallet/test/wallet_test_fixture.h

```diff
@@ -19,10 +19,13 @@ namespace wallet {
 /** Testing setup and teardown for wallet.
  */
 struct WalletTestingSetup : public TestingSetup {
-    explicit WalletTestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
+    WalletTestingSetup() : WalletTestingSetup(CBaseChainParams::MAIN) { }
+    explicit WalletTestingSetup(const std::string& chainName);
+    WalletTestingSetup(const WalletTestingSetup&) = delete;
+    WalletTestingSetup(WalletTestingSetup&&) = delete;
```

Comment:
The changes look unrelated? Also, anything with a unique_ptr member should have them deleted already, before and after the changes here.